serverless.yml

```diff
@@ -91,25 +91,51 @@ resources:
           - Ref: snsTopic
     S3BucketTriggering:
       Type: AWS::S3::Bucket
+      DependsOn:
+        - triggerQueue
+        - triggerQueuePolicy
       Properties:
         BucketName: iow-retriever-capture-${self:custom.environments.${self:provider.stage}}
         AccessControl: Private
         LifecycleConfiguration:
           Rules:
             - ExpirationInDays: 1
               Status: Enabled
-    AqtsCaptureTriggerLambdaPermissionTriggeringS3:
-      Type: 'AWS::Lambda::Permission'
+        NotificationConfiguration:
+          QueueConfigurations:
+            - Event: s3:ObjectCreated:Put
+              Filter:
+                S3Key:
+                  Rules:
+                    - Name: suffix
+                      Value: .json
+              Queue:
+                Fn::GetAtt:
+                  - triggerQueue
+                  - Arn
+    triggerQueue:
+      Type: AWS::SQS::Queue
+      Properties:
+        QueueName: aqts-capture-trigger-queue-${self:provider.stage}
+        MessageRetentionPeriod: 172800
```

Comment:
Just out of curiosity - what is the significance of the `172800` duration.  From the doc it looks like that is the number of seconds to retain a message or ~ 2 days.

Reply:
I wanted to messages to last at least a long as the files in S3 (1 - 2 days).

Reply:
Roger that - thanks!

Reply:
Should we consider a longer retention on prod? Enough to allow us to fix errors that happen on a long weekend?

Reply:
That would probably be a could idea. The retention in the S3 bucket would need to be increased to match so the initial data would be kept too.

Reply:
I've increased it to four days.